### PR TITLE
Caching lfs

### PR DESCRIPTION
**Pull request by Ayato77**
#### Description
- caching LFS is added to actions
- disable lfs by setting all lfs files as excluded

close united-manufacturing-hub/MgmtIssues/issues/1637

### CODE REVIEW

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]
Can you take a look at @Scarjit original comment? I think you kind of missed the original idea here. https://github.com/united-manufacturing-hub/MgmtIssues/issues/1632#issuecomment-1954438351

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM gitpod/workspace-full
```
Comment (Scarjit) [inline .gitpod.Dockerfile:1]: Where did the rest of the .gitpod.Dockerfile go ?

https://github.com/united-manufacturing-hub/ManagementConsole/blob/staging/.gitpod.Dockerfile

Comment (Ayato77) [reply .gitpod.Dockerfile:1]: There was no `.gitpod.Dockerfile` in this repo. Does this repo also need the rest of configuration like https://github.com/united-manufacturing-hub/ManagementConsole/blob/staging/.gitpod.Dockerfile?

Comment (Ayato77) [reply .gitpod.Dockerfile]: nvm, I added the rest of configuration.

**Review by Scarjit — APPROVED** [review]

**Comment by Ayato77** [comment]
@JeremyTheocharis Can you review this PR?

**Pull request closed without merging**